libensemble/gen_funcs/persistent_aposmm.py

```diff
@@ -667,14 +673,15 @@ def add_k_sample_points_to_local_H(k, user_specs, persis_info, n, comm, local_H,
     return persis_info
 
 
-def clean_up_and_stop(local_H, local_opters, run_order):
+# def clean_up_and_stop(local_H, local_opters):
+def clean_up_and_stop(local_opters):
```

Comment:
@jmlarson1 do you still want the option to print local_H information here?

Reply:
I'm fine with removing that (as I can print where `clean_up_and_stop` is called).

Reply:
Ok, be aware, I have moved it to a finally clause, so it is called on exit, including when there are exceptions.